src/main/resources/api/service_openapi.yaml

```diff
@@ -234,6 +257,9 @@ components:
     QueryResponseData:
       type: object
       properties:
+        query_id:
```

Comment:
Why is this needed in the response? The caller already has it, since they needed it to make the API that returned it.

Reply:
I took this back out for now.   My original thought was including as an anchor for maybe pasting between a programmatic output and back into the swagger.

Reply:
Is this taken out or not? It looks like it's still here in the in the yaml.